pymc4/random_variables/random_variable.py

```diff
@@ -108,30 +108,39 @@ class RandomVariable(WithBackendArithmetic):
     calculating the log_prob, while the base distribution is used for sampling
     purposes.
     """
-
+    _bijector = bijectors.Identity()
     _base_dist = None
 
     def __init__(self, name: str, *args, **kwargs):
         self._parents = []
-        self._distribution = self._base_dist(name=name, *args, **kwargs)
+        self._untransformed_distribution = self._base_dist(name=name,
+                                                           *args, **kwargs)
         self._sample_shape = ()
         self._dim_names = ()
         self.name = name
         ctx = contexts.get_context()
         self._creation_context_id = id(ctx)
         self._backend_tensor = None
+        if "bijector" in kwargs.keys():
```

Comment:
Being picky here but you could change this to self._bijector = kwargs.get("bijector",  _bijector)

Reply:
we could do `self._bijector = kwargs.get("bijector", self._bijector)` if that's what you meant.

Reply:
Yea that :)